packages/react-reconciler/src/ReactFiberLane.new.js

```diff
@@ -85,38 +85,54 @@ export const SyncLane: Lane = /*                        */ 0b0000000000000000000
-
-const NonIdleLanes = /*                                 */ 0b0000111111111111111111111111111;
-
-export const IdleHydrationLane: Lane = /*               */ 0b0001000000000000000000000000000;
-const IdleLanes: Lanes = /*                             */ 0b0110000000000000000000000000000;
+const InputDiscreteLane: Lanes = /*                     */ 0b0000000000000000000000000001000;
+
+const InputContinuousHydrationLane: Lane = /*           */ 0b0000000000000000000000000010000;
+const InputContinuousLane: Lanes = /*                   */ 0b0000000000000000000000000100000;
+
+export const DefaultHydrationLane: Lane = /*            */ 0b0000000000000000000000001000000;
+export const DefaultLane: Lanes = /*                    */ 0b0000000000000000000000010000000;
+
+const TransitionHydrationLane: Lane = /*                */ 0b0000000000000000000000100000000;
+const TransitionLanes: Lanes = /*                       */ 0b0000000000000111111111000000000;
+const TransitionLane1: Lane = /*                        */ 0b0000000000000000000010000000000;
+const TransitionLane2: Lane = /*                        */ 0b0000000000000000000100000000000;
+const TransitionLane3: Lane = /*                        */ 0b0000000000000000001000000000000;
+const TransitionLane4: Lane = /*                        */ 0b0000000000000000010000000000000;
+const TransitionLane5: Lane = /*                        */ 0b0000000000000000100000000000000;
+const TransitionLane6: Lane = /*                        */ 0b0000000000000001000000000000000;
+const TransitionLane7: Lane = /*                        */ 0b0000000000000010000000000000000;
+const TransitionLane8: Lane = /*                        */ 0b0000000000000100000000000000000;
+const TransitionLane9: Lane = /*                        */ 0b0000000000001000000000000000000;
+const TransitionLane10: Lane = /*                       */ 0b0000000000010000000000000000000;
+const TransitionLane11: Lane = /*                       */ 0b0000000000100000000000000000000;
+const TransitionLane12: Lane = /*                       */ 0b0000000001000000000000000000000;
+const TransitionLane13: Lane = /*                       */ 0b0000000010000000000000000000000;
+const TransitionLane14: Lane = /*                       */ 0b0000000100000000000000000000000;
+
+const RetryLanes: Lanes = /*                            */ 0b0001111000000000000000000000000;
+const RetryLane1: Lane = /*                             */ 0b0000001000000000000000000000000;
+const RetryLane2: Lane = /*                             */ 0b0000010000000000000000000000000;
+const RetryLane3: Lane = /*                             */ 0b0000100000000000000000000000000;
+const RetryLane4: Lane = /*                             */ 0b0001000000000000000000000000000;
+
+export const SomeRetryLane: Lane = RetryLane1;
+
+export const SelectiveHydrationLane: Lane = /*          */ 0b0001000000000000000000000000000;
```

Comment:
Is this supposed to be the same as RetryLane4?

Reply:
Check again. It was right all along.